docs/Import-Gene-Sets.md

```diff
@@ -32,7 +32,7 @@ Note: This removes existing gene set, gene set hierarchy and gene set genetic pr
 ```
 ./importGenesetData.pl \
 	--data ../../test/resources/genesets/study_es_0_genesets.gmt \
-	--new-version 1 \
+	--new-version msigdb_6.1 \
 	--supp ../../test/resources/genesets/study_es_0_supp-genesets.txt
 ```
 
```

Comment:
I think we should put here that study_es_0 needs gene panels to be loaded, too.

Reply:
This doc is specific for loading gene sets. To load study_es_0, including gene panel data, the user can use https://github.com/thehyve/cbioportal/blob/eb1fccb23fecf271a3e9ba81f463b4dbb5df6904/docs/Load-Sample-Cancer-Study.md